apps/trouble-host-unavailable.html.markerb

```diff
@@ -0,0 +1,83 @@
+---
+title: Troubleshoot apps when a host is unavailable
+objective: 
+layout: docs
+nav: firecracker
+---
+
+Fly.io has multiple hosts (physical hardware) in every supported region. A host could be unavailable because of a hardware failure, connectivity issues, or other reasons. A host issue might result in a complete or intermittent outage, or generally degraded performance for apps and databases without redundancy. A host could be down for an hour or a day, or sometimes longer. We understand that this can be frustrating if your app doesn’t have extra Machines or replicated databases for redundancy.
+
+For production apps, or any apps that require high availability, we strongly recommend running multiple machines and volumes for redundancy in the case of a single host failure. This goes double for apps with volumes, as volumes are pinned to specific hosts.
```

Comment:
```suggestion
Fly.io has multiple hosts (physical servers) in every supported [region](/docs/reference/regions/). From time to time, a hardware failure, connectivity issues, or other factor will make a host unavailable. A single-host issue may cause a sustained or intermittent outage, or generally degraded performance, and this will disproportionately affect apps that run on a single Machine.

A host can be down for an hour, or a day, or sometimes longer. 

For production apps, or any apps that require high availability, we strongly recommend redundancy as a buffer against single-host failure. This means running multiple Machines, and for stateful apps it means having a clustering mechanism (such as those built into LiteFS and Postgres, for example) to replicate data between multiple volumes. as volumes are pinned to specific hosts.
```

Take anything you like from here